lib/cli/in.js

```diff
@@ -14,23 +27,20 @@ var fs = require('fs'),
  * @param {String} url - The location of the remote resource
  * @param {Function} callback
  */
-var fetchResource = function(url, callback) {
-	logger('Fetching URL resource: ' + url);
+var fetchResource = function (url, callback) {
+  logger("Fetching URL resource: " + url);
 
-	request(url, function (error, response, body) {
-		if (!error && response.statusCode == 200) {
-			callback(body);
-		} else {
-			var errorString = [
-				'There was an error loading the requested resource',
-				'>>> ' + url
-			];
-			if (error) {
-				errorString.push(error.toString('utf8'));
-			}
-			errorHandler(errorString.join('\n'));
-		}
-	});
+  fetch(url)
+    .then(function (response) {
+      if (response.status === 200) {
+        callback(body);
```

Comment:
Good call - I'll fix that up. I guess I'm too used to Typescript where I would have gotten a type error 😁

Reply:
Should be fixed now! I also spent a lot of time making sure the unit tests actually worked - they would pass previously but they weren't actually working due to the way stdout and stderr were being passed back in the test helper function. I added in a test express server that would allow us to verify that fetch is bringing back the correct data.